CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 ### AppCenterAnalytics
 
+* **[Feature]** Adding support for typed properties. Note that these APIs still convert properties back to strings on the App Center backend. More work is needed to store and display typed properties in the App Center portal. Using the new APIs now will enable future scenarios, but for now the behavior will be the same as it is for current event properties.
```

Comment:
@elamalani how should we advertise this for 3p customers since it's not yet useful for them?

Reply:
got offline confirmation